controllers/k8sgpt_controller.go

```diff
@@ -159,7 +164,7 @@ func (r *K8sGPTReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 			// Create a new client
 			var address string
 			if os.Getenv("LOCAL_MODE") != "" {
-				address = "localhost:8080"
+				address = "localhost:8082"
```

Comment:
I imagine this was just for debugging with something else running locally?

Reply:
they slipped in..